### ISSUE TITLE
Cannot use cache.use_sliding_expiration in hibernate.cfg.xml

### ISSUE THREAD

**Issue by boypula** [nhibernate/nhibernate-core#1650]
When I include `<property name="cache.use_sliding_expiration">true</property>` in a hibernate.cfg.xml file, I received the following exception:

`An exception occurred parsing configuration :The 'name' attribute is invalid - The value 'cache.use_sliding_expiration' is invalid according to its datatype 'String' - The Enumeration constraint failed.`

When I checked the source, **cache.use_sliding_expiration** is not defined in nhibernate-configuration.xsd

**Comment by fredericDelaporte** [comment]
Indeed, this feature added with nhibernate/NHibernate-Caches#12 has been tested only by programmatic configuration, and updating `nhibernate-configuration.xsd` has been overlooked.

A workaround is to define this configuration property directly on the `NHibernate.Cfg.Configuration` object used to build the session factory. (`cfg.SetProperty("cache.use_sliding_expiration", "true");`)

@hazzik, does this really deserve a 5.1.2 patch? The caches configuration was indeed broken with e542bdfe04bef4b362f9868244c57202992fae27, when the `name` attribute of `property` in xsd was changed to an enumeration. Many caches were relying on custom configuration properties which were not added to the enumeration list.

Adding them all would quite bloat that list I think. This pattern of using NHibernate configuration properties for an external library like NHibernate.Caches should be avoided in fact. All caches parameters should be handled by their own configuration section instead.

But at the time I have implemented nhibernate/NHibernate-Caches#12, I have just stick to the old way most caches were configured for their global settings, relying on NHibernate configuration properties. (And being "configurable" only programmatically for all parameters "missing" in the xsd, like the Bamboo Prevalence base path or the SysCache default priority.)

So why not adding this `cache.use_sliding_expiration` property in the xsd since it is now used by many caches implementation. But as the situation is not really new and has a workaround, I think it can be done in 5.2 rather than doing a 5.1.2 release for that.
